indico/modules/events/surveys/templates/emails/new_submission_email.txt

```diff
@@ -1,27 +1,35 @@
-{% extends 'emails/base.txt' %}
+{% extends 'emails/base_i18n.txt' %}
 {% set survey = submission.survey %}
 
 {% block subject -%}
-    New survey submission for "{{ survey.title }}" on the event "{{ survey.event.title }}"
+    {% trans sur=survey.title, evt=survey.event.title %}New survey submission for "{{ sur }}" on the event "{{ evt }}"{% endtrans %}
```

Comment:
I'd use meaningful names like `survey_title` and `event_title` - easier for translators if they know what a placeholder is used for. Same in the other survey templates which currently use the same abbreviations